site/docs/components/multiline-input/accessibility.mdx

```diff
@@ -8,73 +8,75 @@ data:
   $ref: ./#/data
 ---
 
-- Wrap Multiline Input in a [Form Field](/salt/components/form-field) to make it accessible within a form. This will provide the field with a visible label, help text and a status message for validation feedback. More information can be found [in the W3C form instructions](https://www.w3.org/WAI/tutorials/forms/instructions/).
+### Best practices
 
-- Text that extends beyond the visible area of Multiline Input is clipped. This behavior is common to all states, i.e., default, read-only and disabled—whether or not the input has focus. To view the text in its entirety, the user can traverse the characters using the cursor.
+- Wrap `MultilineInput` in [`FormField`](/salt/components/form-field) to make it accessible within a form. This will provide the field with a visible label, help text and a status message for validation feedback. You can find more information [in the W3C form instructions](https://www.w3.org/WAI/tutorials/forms/instructions/).
```

Comment:
I know this is in Input too but make it `accessible` within a form is technically incorrect and is a bit confusing.

Reading this I think of two possibilities:

`MultiliineInput`'s value cannot be accessed within a form if it's not wrapped in `FormField` which is incorrect.
`MultilineInput` won't meet accessibility guidance within a form if it's not wrapped in a `FormField` which is partly true. To be considered compliant with accessibility guidelines a field needs a label which is what `FormField` does but this is irrespective of whether it's in a form or not.